kits/democracy/contracts/DemocracyKit.sol

```diff
@@ -51,5 +53,10 @@ contract DemocracyTemplate is BetaTemplateBase {
             minAcceptanceQuorum,
             voteDuration
         );
+
+        // burn support modification permission
+        ACL acl = ACL(Kernel(voting.kernel()).acl());
+        acl.createPermission(NO_ENTITY, voting, voting.MODIFY_SUPPORT_ROLE(), NO_ENTITY);
```

Comment:
Setting the manager to `NO_ENTITY` doesn't burn the permission as they can be re-created when the manager is `0x0`

Reply:
Oh, yes, you are right, thanks! What do you think it's the best way to accomplish that? Using something else different than `NO_ENTITY` and `ANY_ENTITY` like for instance `address(1)`? Is there a better way? Maybe we could add a function in `ACL.sol` called `burnPermission` to make it easier and more clear.

Reply:
I think the least confusing is going to be adding it to the ACL. Maybe rather than `burnPermission` we can call it `lockPermission`, `freezePermission` or `burnPermissionManager` as if someone already had the permission they will be able to continue performing the actions forever, as there will never be a manager again.

Actually `petrifyPermission` could be a good name as well 😅 Paging @sohkai for his input on this too.